functions.yml

```diff
@@ -187,26 +201,23 @@ etl:
           doc: Location of the chain file on each node in the cluster
           type: str
         - name: minMatchRatio
-          doc: Minimum fraction of bases that must remap to lift over successfully
+          doc: Minimum fraction of bases that must remap to do liftover successfully. If not provided, defaults to 0.95.
           type: double
           is_optional: true
+      returns: A struct containing ``contigName``, ``start``, and ``end`` fields after liftover.
     - name: normalize_variant
       doc: |
-             Normalize the variant with a behavior similar to vt normalize or bcftools norm.
+             Normalizes the variant with a behavior similar to vt normalize or bcftools norm.
              Creates a StructType column including the normalized start, end, referenceAllele and
              alternateAlleles fields (whether they are changed or unchanged as the result of
              normalization) as well as a StructType field called normalizationStatus that
              contains the following fields:
 
-             changed: A boolean field indicating whether the variant data was changed as a
-                 result of normalization.
+                changed: A boolean field indicating whether the variant data was changed as a result of normalization.
 
-             errorMessage: An error message in case the attempt at normalizing the row hit an
-                 error. In this case, the changed field will be set to false. If no errors occur,
-                 this field will be null.
+                errorMessage: An error message in case the attempt at normalizing the row hit an error. In this case, the changed field will be set to false. If no errors occur, this field will be null.
```

Comment:
Wrap `errorMessage` in backticks. We probably don't need to wrap generic types, but we should wrap any values (eg. `false`, `null`)

Reply:
Done